TestModels/SimpleTypes/SimpleBlob/runtimes/rust/tests/simple_blob_test.rs

```diff
@@ -0,0 +1,81 @@
+use simple_blob::*;
+
+/*
+        method{:test} GetBlob(){
+        var client :- expect SimpleBlob.SimpleBlob();
+        TestGetBlob(client);
+        TestGetBlobKnownValueTest(client);
+    }
+*/
+
+/*
+   method TestGetBlob(client: ISimpleTypesBlobClient)
+     requires client.ValidState()
+     modifies client.Modifies
+     ensures client.ValidState()
+   {
+       var s: seq<UInt.uint8> := [0x0, 0x1, 0x2];
+       var convertedBlobInput: GetBlobInput := SimpleBlob.Types.GetBlobInput(value:= Some(s));
+       SimpleBlobImpl.ValidateBlobType(convertedBlobInput.value.value);
+       // Validate values of convertedBlobInput are same as input values
+       expect convertedBlobInput.value.value == s;
+
+       var ret :- expect client.GetBlob(convertedBlobInput);
+
+       // Expect output of GetBlob has same value as input sequence
+       // i.e. GetBlob(GetBlobInput(seq)) == GetBlobInput(seq)
+       expect ret.value.UnwrapOr([0x0]) == convertedBlobInput.value.value;
+       // From above, GetBlobInput(seq) value == seq, so the below should be redundant
+       expect ret.value.UnwrapOr([0x0]) == s;
+       print ret;
+   }
+*/
+#[tokio::test]
+async fn test_get_blob() {
+    let s = ::dafny_runtime::Sequence::from_array(&vec![0x0, 0x1, 0x2]);
```

Comment:
This is the primary clue the types aren't right, because these tests shouldn't refer to any Dafny types.